test/models/category_test.rb

```diff
@@ -1,7 +1,14 @@
 require 'test_helper'
-
 class CategoryTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    @category = categories(:one)
+  end
+
+  test "no_destroy should return false" do
+    refute @category.no_destroy
+  end
+
+  test "to_s must return category name" do
+    assert_equal @category.name, @category.to_s, "to_s failed and is not equal to name"
```

Comment:
Shouldn't need the third option, the default message from `assert_equal` should be fine